.github/workflows/release.yml

```diff
@@ -0,0 +1,33 @@
+name: Release builder
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  release_build:
+    runs-on: ${{ matrix.operating-system }}
+    strategy:
+      matrix:
+        operating-system: [ubuntu-latest]
+        php-versions: ['7.3']
+    steps:
+    - name: Install PHP ${{ matrix.php-versions }}
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: ${{ matrix.php-versions }}
+    - name: "Checkout repository"
+      uses: actions/checkout@master
+    - name: "Composer install"
+      uses: MilesChou/composer-action/8.1@master
+      with:
+        args: install --ignore-platform-req=ext-* --no-dev
```

Comment:
- do we need `shivammathur/setup-php`? (Composer.lock is for PHP7.3)
- `MilesChou/composer-action` could be replaced with code from other workflows

Reply:
right, done!